kurtosis-module/impl/participant_network/cl/lodestar/lodestar_launcher.go

```diff
@@ -184,37 +189,37 @@ func (launcher *LodestarClientLauncher) getBeaconContainerConfig(
 		"--logLevel=" + logLevel,
 		fmt.Sprintf("--port=%v", discoveryPortNum),
 		fmt.Sprintf("--discoveryPort=%v", discoveryPortNum),
-		"--rootDir=" + consensusDataDirpathOnServiceContainer,
+		"--dataDir=" + consensusDataDirpathOnServiceContainer,
 		"--paramsFile=" + genesisConfigFilepath,
 		"--genesisStateFile=" + genesisSszFilepath,
 		"--eth1.depositContractDeployBlock=0",
 		"--network.connectToDiscv5Bootnodes=true",
-		"--network.discv5.enabled=true",
-		"--eth1.enabled=true",
+		"--discv5=true",
+		"--eth1=true",
 		"--eth1.providerUrls=" + elClientRpcUrlStr,
 		"--execution.urls=" + elClientEngineRpcUrlStr,
-		"--api.rest.enabled=true",
-		"--api.rest.host=0.0.0.0",
-		"--api.rest.api=*",
-		fmt.Sprintf("--api.rest.port=%v", httpPortNum),
+		"--rest=true",
+		"--rest.address=0.0.0.0",
+		"--rest.namespace=*",
+		fmt.Sprintf("--rest.port=%v", httpPortNum),
 		"--enr.ip=" + privateIPAddressPlaceholder,
 		fmt.Sprintf("--enr.tcp=%v", discoveryPortNum),
 		fmt.Sprintf("--enr.udp=%v", discoveryPortNum),
 		// Set per Pari's recommendation to reduce noise in the logs
-		"--network.subscribeAllSubnets=true",
+		"--subscribeAllSubnets=true",
 		fmt.Sprintf("--jwt-secret=%v", jwtSecretFilepath),
 		// vvvvvvvvvvvvvvvvvvv METRICS CONFIG vvvvvvvvvvvvvvvvvvvvv
-		"--metrics.enabled",
-		"--metrics.listenAddr=0.0.0.0",
-		fmt.Sprintf("--metrics.serverPort=%v", metricsPortNum),
+		"--metrics",
+		"--metrics.address=0.0.0.0",
+		fmt.Sprintf("--metrics.port=%v", metricsPortNum),
```

Comment:
I guess all those changes plays a role in the fact that now the clients don't do PoW anymore. But I don't see their version being updated anywhere in this PR, so I'm wondering how that works. Is it because we're pulling `latest` systematically for all of them?

Reply:
The changes for lodestar are related to their release from ~1mo ago, they changed a lot of their flags to follow a similar convention as other clients. So any version that's recent should accept the new flags.

Reply:
Got it, that makes sense